runner/runner/listener.py

```diff
@@ -21,51 +22,43 @@ def start_listening():
         connection.ioloop.start()
 
 
-def _on_connected(connection):
+def _on_connection_open(connection):
     """Called when we are fully connected to RabbitMQ"""
-    logger.debug("connected...")
+    logger.info("Connected")
     connection.channel(on_open_callback=_on_channel_open)
 
 
 def _on_channel_open(new_channel):
     """Called when our channel has opened"""
-    logger.debug("channel opened...")
-    global channel
-    channel = new_channel
-    channel.queue_declare(
```

Comment:
I removed the queue declaration stuff here because the runner shouldn't be deciding and creating that stuff.  It should just be able to assume that the queue it's going to listen on has already been created for it.  For now, that's the public runner queue.  Eventually, that'll be a queue established by the main app during runner registration.